data/cwisep_j181006.00-101001.1.json

```diff
@@ -0,0 +1,77 @@
+{
+    "Sources": [
+        {
+            "source": "CWISEP J181006.00-101001.1",
+            "ra": 272.52574999999996,
+            "dec": -10.166805555555555,
+            "epoch": null,
+            "equinox": null,
+            "shortname": null,
+            "reference": "Schn20",
+            "other_references": null,
+            "comments": null
+        }
+    ], 
+    "ModeledParameters": [
+        {
+            "parameter": "log g",
+            "value": 5.0,
+            "value_error": 0.25,
+            "unit": "dex",
+            "comments": null,
+            "reference": "Lodi22"
+        },
+        {
+            "parameter": "mass",
+            "value": 17.0,
+            "value_error": 56.0,
+            "unit": "M_jup",
+            "comments": null,
+            "reference": "Lodi22"
+        },
+        {
+            "parameter": "metallicity",
+            "value": -1.5,
+            "value_error": 0.5,
+            "unit": "dex",
+            "comments": null,
+            "reference": "Lodi22"
+        },
+        {
+            "parameter": "radius",
+            "value": 0.067,
+            "value_error": 0.032,
+            "unit": "R_jup",
```

Comment:
The value in the paper is actually in solar radaii, not jupiter radii.  You need to convert. I recommend using astropy units. https://docs.astropy.org/en/stable/constants/.  https://docs.astropy.org/en/stable/units/quantity.html#quantity-unit-conversion

Reply:
0.067 and 0.032 needs to be in solar radii, which need to be coded into the json file with astropy?

Reply:
Try just changing the unit.  R_sun instead of R_jup.